examples/energy-management-app/energy-management-common/device-energy-management/src/DEMTestEventTriggers.cpp

```diff
@@ -28,19 +29,22 @@ using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::DeviceEnergyManagement;
+namespace {
 
-static constexpr uint16_t MAX_SLOTS             = 10;
-static constexpr uint16_t MAX_POWER_ADJUSTMENTS = 5;
+constexpr uint16_t MAX_SLOTS             = 10;
+constexpr uint16_t MAX_POWER_ADJUSTMENTS = 5;
 
-static chip::app::Clusters::DeviceEnergyManagement::Structs::SlotStruct::Type sSlots[MAX_SLOTS];
-static chip::app::Clusters::DeviceEnergyManagement::Structs::ForecastStruct::Type sForecastStruct;
-static chip::app::DataModel::Nullable<chip::app::Clusters::DeviceEnergyManagement::Structs::ForecastStruct::Type> sForecast;
+chip::app::Clusters::DeviceEnergyManagement::Structs::SlotStruct::Type sSlots[MAX_SLOTS];
```

Comment:
OOC why remove the static keyword?

Reply:
In this case, static is redundant, because these variables are in an anonymous namespace which limits the visibility of definitions to a single translation unit.